backend/routes/indexer/nft.go

```diff
@@ -135,6 +136,36 @@ func processNFTMintedEvent(event IndexerEvent) {
 		return
 	}
 
+	// Create a NFT JSON metadata file
+	metadata := map[string]interface{}{
+		"name":        fmt.Sprintf("NFT #%d", tokenId),
+		"description": "This is an NFT minted on the blockchain.",
+		"image":       fmt.Sprintf("ipfs://%s", imageHashHex),
+		"attributes": []map[string]string{
+			{
+				"trait_type": "Width",
+				"value":      fmt.Sprintf("%d", width),
+			},
+			{
+				"trait_type": "Height",
+				"value":      fmt.Sprintf("%d", height),
+			},
+		},
```

Comment:
An attribute for Position ( x, y ) would be cool, also the minter's name or address.